specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/blob.json

```diff
@@ -937,6 +977,11 @@
           "type": "boolean",
           "readOnly": true,
           "description": "The hasImmutabilityPolicy public property is set to true by SRP if ImmutabilityPolicy has been created for this container. The hasImmutabilityPolicy public property is set to false by SRP if ImmutabilityPolicy has not been created for this container."
+        },
+        "versionLevelWorm": {
+          "$ref": "#/definitions/VersionLevelWorm",
+          "x-ms-client-flatten": true,
```

Comment:
are you sure to flatten all properties for this model? With flatten `true`, all sub-properties including `enabled` are flattened without `versionLevelWorm` info as you can see below:
![image](https://user-images.githubusercontent.com/49508232/111270107-655a3c80-866a-11eb-8c52-b1519b846fb0.png)